bundles/admin/admin-layereditor/view/LocalizingFlyout.jsx

```diff
@@ -158,10 +185,17 @@ const LocalizedContent = ({ loading, labels, value, headerMessageKey, controller
             >
                 <TextInput />
             </LocalizationComponent>
+            {id && layerCountInGroup > 0 &&
+                <DeleteLayersCheckbox checked={deleteLayers} onChange={ evt => controller.setDeleteLayers(evt.target.checked)}>{deleteMapLayersText + ' (' + layerCountInGroup + ')'}</DeleteLayersCheckbox>
```

Comment:
We should add a confirmation popup for delete and add this as an option to the confirmation. Now it's a bit scary looking field in the form if the user just wants to update the name. It could also be just 2 different buttons on the popup. One that deletes the provider including layers and one that only removes the provider and moves layers to the "ungrouped" group.

Reply:
Confirmation popup is now added.